src/openfermion/_version.py

```diff
@@ -11,4 +11,4 @@
 #   limitations under the License.
 
 """Define version number here and read it from setup.py automatically"""
-__version__ = "1.6.0"
+__version__ = "1.6.0.dev"
```

Comment:
this should be 1.7.0.dev0 probably

Reply:
I guess, how about 1.6.1.dev0 as I imagine releasing some bug fixes for #839 etc (what is the 0 after the dev for, it seems cirq doesn't have the zero, qualtran does)

Reply:
My reading of the spec: https://peps.python.org/pep-0440/#public-version-identifiers says that it should have a zero. https://peps.python.org/pep-0440/#developmental-releases but everything will work without it https://peps.python.org/pep-0440/#implicit-development-release-number

If we were to do things the right way :tm: patch releases would be made off of a branch. E.g. `master` would move to 1.7.0.dev0; fixes are committed there; and critical bugfixes are cherry-picked to a `1.6` branch that branches off of the tagged release